.circleci/generate_config.py

```diff
@@ -351,7 +368,42 @@ def add_test_definition_jobs_to_workflow(
             jobs.append(create_test_job(test, False, build_config, build_job))
 
 
-def add_test_jobs_to_workflow(workflow, tests, build_config, build_job, repl2):
+def add_rta_ui_test_jobs_to_workflow(args, workflow, build_config, build_job):
+    jobs = workflow["jobs"]
+    ui_testsuites = [
+        "UserPageTestSuite",
+        "CollectionsTestSuite",
+        "ViewsTestSuite",
+        "GraphTestSuite",
+        "QueryTestSuite",
+        "AnalyzersTestSuite",
+        "DatabaseTestSuite",
+        "LogInTestSuite",
+        "DashboardTestSuite",
+        "SupportTestSuite",
+        "ServiceTestSuite",
+    ]
+    if args.ui_testsuites != "":
+        ui_testsuites = args.ui_testsuites.split(',')
+    deployments = ['SG'
+                   #"CL",
+                   ]
+    if args.ui_deployments:
+        deployments = args.ui_deployments.split(',')
+
+    for deployment in deployments:
+        for test_suite in ui_testsuites:
+            jobs.append(create_rta_test_job(build_config, build_job, deployment, test_suite))
+
+
+def add_test_jobs_to_workflow(args, workflow, tests, build_config, build_job, repl2):
+    if build_config.arch == "x64" and args.ui != "" and args.ui != "off":
+        if build_config.enterprise:
+            add_rta_ui_test_jobs_to_workflow(args, workflow, build_config, build_job)
+        elif args.ui == "community":
+            add_rta_ui_test_jobs_to_workflow(args, workflow, build_config, build_job)
+        if args.ui == "only":
```

Comment:
Why do we have logically inequal values within the same `ui` parameter?

Reply:
- we will allways run enterprise.
- if `community` we will those as well. 
- if `only` we won't run `testing.js` tests that would be added below.

Reply:
Questions/suggestions

1. What is `testing.js`?
2. what do you think about making `ui` an array, which can have any of these values: `'enterprise' | 'community' | 'ARM' | 'testing.js'`. So if we want to run both, we will set it to `['enterprise', 'community']`.

Reply:
`testing.js` is the framework which runs all the regular integration tests. Its using arangosh. 
The selenium tests are created under the python written RTA framework, which attempts to test installations and deployments - we use it during the release testing.
Regarding ARM, I don't think that running the tests on ARM yields different results, unless maybe we run them with instrumented binaries (which then tend to be slooouuwww)